composer.json

```diff
@@ -53,7 +53,7 @@
         "ext-xml": "*",
         "ext-zip": "*",
         "ext-zlib": "*",
-        "aws/aws-sdk-php": "~3.198.5",
+        "aws/aws-sdk-php": "^3.198.5",
```

Comment:
you need to change src/Core/composer.json too

Reply:
@shyim True. done